File: 0006.c
#include <stdio.h>

void fn0() {
  int i;
  long int sumofsq = 0;
  long int sum = 0;
  for (i = 1; i < 101; i++) {
    sumofsq += (i * i);
    sum += i;
  }
  printf("%d\n", sum * sum - sumofsq);
}

void (*solutions[])() = { fn0, NULL };

File: 0006.java
public class _0006 {

    public Object fn0() {
        int i;
        long sumofsq = 0;
        long sum = 0;
        for (i = 1; i < 101; i++) {
            sumofsq += (i * i);
            sum += i;
        }
        return sum * sum - sumofsq;
    }

    public String[] solutions = new String[]{"fn0"};
}
